examples/resources/awscc_securityhub_configuration_policy/main.tf

```diff
@@ -0,0 +1,48 @@
+# Get current AWS account ID
+data "aws_caller_identity" "current" {}
+
+# Get current AWS region
+data "aws_region" "current" {}
```

Comment:
These don't appear to be used

Reply:
Noticed it as well. I am going to review the cleanup step we have defined to check for any of these unused datasources and revisit all of these examples.